runtime/oti/j9modifiers_api.h

```diff
@@ -82,8 +82,12 @@
 #define J9ROMCLASS_IS_CONTENDED(romClass)	_J9ROMCLASS_J9MODIFIER_IS_SET((romClass), J9AccClassIsContended)
 
 #ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
-/* Will need to modify this if ValObject/RefObject proposal goes through */
+/*
+ * TODO: Will need to modify this if ValObject/RefObject proposal goes through.
+ * Some exiting places using J9ROMCLASS_IS_VALUE() may need to check J9ROMCLASS_IS_PRIMITIVE().
+ */
 #define J9ROMCLASS_IS_VALUE(romClass)	_J9ROMCLASS_SUNMODIFIER_IS_SET((romClass), J9AccValueType)
+#define J9ROMCLASS_IS_PRIMITIVE(romClass)	_J9ROMCLASS_SUNMODIFIER_IS_SET((romClass), J9AccPrimitive)
```

Comment:
please name to `J9ROMCLASS_IS_PRIMITIVE_VALUE_TYPE` to avoid confusion with existing primitives

Reply:
Fixed.